rules/ast/functions.go

```diff
@@ -291,6 +291,25 @@ func dropCommonLabelsImpl(timestamp clientmodel.Timestamp, args []Node) interfac
 	return vector
 }
 
+// === round(vector VectorNode) Vector ===
+func roundImpl(timestamp clientmodel.Timestamp, args []Node) interface{} {
+	// round to even
```

Comment:
Ok, my understanding is that I'll add commits until I receive a LGTM, then I'll squash and force push and you'll merge. Is this correct?